pom.xml

```diff
@@ -4,6 +4,13 @@
   <modelVersion>4.0.0</modelVersion>
   <version>${revision}</version>
 
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>2.7.1</version>
+    <relativePath/>
+  </parent>
+
   <groupId>com.company</groupId>
```

Comment:
rename group, for example com.sabetski.edi, and use this one as your core package while java code writing